gateway/ln-gateway/src/rpc/rpc_server.rs

```diff
@@ -178,7 +178,7 @@ fn v1_routes(gateway: Arc<Gateway>) -> Router {
     // authenticated after a password has been set.
     let authenticated_after_config_routes = Router::new()
         .route(SET_CONFIGURATION_ENDPOINT, post(set_configuration))
-        .route(CONFIGURATION_ENDPOINT, get(configuration))
+        .route(CONFIGURATION_ENDPOINT, post(configuration))
```

Comment:
The usage of `CONFIGURATION_ENDPOINT` on the client side has been as a POST request for a long time - how was this not caught? 😆

Should we add a test for this endpoint to prevent regression?

Reply:
I wasn't aware of any use case for it, and was surprised when I found it there. But this is the endpoint we will need to use from the UI.

Yes, I will add a test.

Reply:
Added a test